FedComm/src/FedComm.cpp

```diff
@@ -1,7 +1,9 @@
 #include "FedComm.hpp"
 #include <glog/logging.h>
 
-
+/*
+Constructor
+*/
 FedCommunication :: FedCommunication()
 {
   fedCommSockfd = -1; 
```

Comment:
hmm.. where can i find the declaration of this variable, it's not there in FedComm.hpp also, am i missing something here ?